samples/go/csv/read.go

```diff
@@ -44,6 +46,31 @@ func KindsToStrings(kinds KindSlice) []string {
 	return strs
 }
 
+//EscapeStructFieldForCSV removes special characters and replaces spaces with camelCasing (camel' case turns to camelCase)
+func EscapeStructFieldFromCSV(input string) string {
+
+	encode := func(s1 string, p *regexp.Regexp) string {
+		if p.MatchString(s1) {
+			return s1
+		}
+		return ""
+	}
+
+	splitFields := strings.Fields(input)
+	output := types.EscapeFields(splitFields[0], encode)
+
+	if len(splitFields) > 1 {
+		output = strings.ToLower(output)
+	}
+
+	for _, field := range splitFields[1:] {
+		output += strings.Title(strings.ToLower(types.EscapeFields(field, encode)))
+	}
+
+	types.IsValidStructFieldName(output)
```

Comment:
Not checking the return value of this. Left over debugging? If not, please check the return value.